config/locales/en.yml

```diff
@@ -21,6 +21,9 @@
 
 en:
   hello: "Welcome to the Workshop-Portal"
+  unauthorized:
+      manage:
+        all: "Not authorized to %{action} %{subject}."
```

Comment:
I think this should be changed to simply "You are not authorized to take this action." or "You are not authorized to view this page." without leaking the subject and action to the user.